tests/src/RunBenchmarks/RunBenchmarks.cs

```diff
@@ -0,0 +1,1049 @@
+            string complusVersion = Controls.ComplusVersion;
+            string benchmarksRootDirectory = Controls.BenchmarksRootDirectory;
+            string benchmarkDirectory = Path.Combine(benchmarksRootDirectory, benchmark.WorkingDirectory);
+            bool doRunInShell = benchmark.DoRunInShell;
+            bool useSSE = benchmark.UseSSE;
+            bool useAVX = benchmark.UseAVX;
+            int expectedResults = benchmark.ExpectedResults;
+            int failureResults = ~expectedResults;
+            int actualResults;
+            int failures = 0;
+
+            Results results = new Results(benchmark, numberOfRuns);
+
+            if (!doRun)
+            {
+                return results;
+            }
+
+            string workingDirectory = benchmarkDirectory;
+            string fileName = Path.Combine(workingDirectory, benchmark.ExeName);
+            string args = benchmark.ExeArgs;
+
+            if (runner != "")
+            {
+                args = fileName + " " + args;
+                fileName = runner;
+            }
+
+            if (doDebugBenchmark)
+            {
+                args = "/debugexe " + fileName + " " + args;
+                fileName = "devenv.exe";
+            }
+
+            if (doRunInShell)
+            {
+                args = "/C " + fileName + " " + args;
+                fileName = "cmd.exe";
+            }
+
+            if (doVerbose)
+            {
+                Console.WriteLine("Running benchmark {0} ...", benchmark.Name);
+                Console.WriteLine("Invoking: {0} {1}", fileName, args);
+            }
+
+            for (int run = (doWarmUpRun) ? 0 : 1; run <= numberOfRuns; run++)
+            {
+                ProcessStartInfo startInfo = new ProcessStartInfo()
+                {
+                    FileName = fileName,
+                    Arguments = args,
+                    WorkingDirectory = workingDirectory,
+                    UseShellExecute = false
+                };
+
+                if (complusVersion != "")
+                {
+                    startInfo.Environment["COMPLUS_version"] = complusVersion;
+                    startInfo.Environment["COMPLUS_defaultVersion"] = complusVersion;
+                }
+                if (useSSE)
+                {
+                    startInfo.Environment["COMPLUS_FeatureSIMD"] = "1";
+                    startInfo.Environment["COMPLUS_EnableAVX"] = "0";
+                }
+                if (useAVX)
+                {
+                    startInfo.Environment["COMPLUS_FeatureSIMD"] = "1";
+                    startInfo.Environment["COMPLUS_EnableAVX"] = "1";
+                }
+                startInfo.Environment["COMPLUS_gcConcurrent"] = "0";
+                startInfo.Environment["COMPLUS_gcServer"] = "0";
+                startInfo.Environment["COMPLUS_NoGuiOnAssert"] = "1";
+                startInfo.Environment["COMPLUS_BreakOnUncaughtException"] = "0";
```

Comment:
That is to say, *nix is case-sensitive. that means using COMPlus_Version and COMPlus_DefaultVersion as well. Might want to double-check the others.